lib/app.js

```diff
@@ -11,6 +11,8 @@ var cachecache = require("cachecache"),
     handlebars = require("handlebars"),
     mercator = new (require("sphericalmercator"))();
 
+var abaculus = require('abaculus');
```

Comment:
Please move this into the alphabetized list of "community" dependencies (the block above).